### ISSUE TITLE
How to run more than one account?

### ISSUE THREAD

**Issue by 13918763630** [shamhi/HamsterKombatBot#2308]
If I have several telegram accounts, do I need to generate the api_ids and api_hashs for each account? If I should do so, how can I fill them? Because in .env i only need provide one api_id and api_hash.

**Comment by afillimonov** [comment]
> If I have several telegram accounts, do I need to generate the api_ids and api_hashs for each account? If I should do so, how can I fill them? Because in .env i only need provide one api_id and api_hash. 

you need to run the bot and in menu you can add as many as you like sessions

**Comment by 13918763630** [comment]
> > If I have several telegram accounts, do I need to generate the api_ids and api_hashs for each account? If I should do so, how can I fill them? Because in .env i only need provide one api_id and api_hash.
> 
> you need to run the bot and in menu you can add as many as you like sessions

But the api_id and api_hash is just generated by the first account. In each session, i can only choose the proxy and log in with phone, is that okay? Using just one api pairs for all of the accounts?

**Comment by afillimonov** [comment]
> > > If I have several telegram accounts, do I need to generate the api_ids and api_hashs for each account? If I should do so, how can I fill them? Because in .env i only need provide one api_id and api_hash.
> > 
> > you need to run the bot and in menu you can add as many as you like sessions
> 
> But the api_id and api_hash is just generated by the first account. In each session, i can only choose the proxy and log in with phone, is that okay? Using just one api pairs for all of the accounts?

yes, you need only one api pairs and add sessions by logging into the account

**Comment by 13918763630** [comment]
> > > > If I have several telegram accounts, do I need to generate the api_ids and api_hashs for each account? If I should do so, how can I fill them? Because in .env i only need provide one api_id and api_hash.
> > > 
> > > 
> > > you need to run the bot and in menu you can add as many as you like sessions
> > 
> > 
> > But the api_id and api_hash is just generated by the first account. In each session, i can only choose the proxy and log in with phone, is that okay? Using just one api pairs for all of the accounts?
> 
> yes, you need only one api pairs and add sessions by logging into the account

Great! Thank you so much!

**Comment by 13918763630** [comment]
> > > > If I have several telegram accounts, do I need to generate the api_ids and api_hashs for each account? If I should do so, how can I fill them? Because in .env i only need provide one api_id and api_hash.
> > > 
> > > 
> > > you need to run the bot and in menu you can add as many as you like sessions
> > 
> > 
> > But the api_id and api_hash is just generated by the first account. In each session, i can only choose the proxy and log in with phone, is that okay? Using just one api pairs for all of the accounts?
> 
> yes you need only one api pairs and add sessions by logging into the account

No matter how many accounts i will use, just one api pairs is enough? Will my tg account be banned due to this issue or be considered as Sybil Attack by the hamster? Lets say for example 1000 accounts (though i don't have so many lol)
